# DS-mRNA
Papers in mRNA therapeutics, with the focus on data science applications, including data sets, machine learning algorithms, and software.

- [DS-mRNA](#ds-mrna)
  - [Overview](#overview)
    - [Clincial trials](#clincial-trials)
  - [mRNA translation and stability](#mrna-translation-and-stability)
    - [Translation quantification](#translation-quantification)
    - [5' untranslated region (UTR)](#5-untranslated-region-utr)
    - [Codon optimization](#codon-optimization)
    - [3' UTR](#3-utr)
  - [mRNA structure](#mrna-structure)
    - [secondary structure](#secondary-structure)
    - [3D structure](#3d-structure)
  - [Delivery technology](#delivery-technology)
  - [mRNA production](#mrna-production)
  - [circular RNA (including cap-independent mRNA translation)](#circular-rna-including-cap-independent-mrna-translation)
    - [Internal ribosome entry site (IRES)](#internal-ribosome-entry-site-ires)

## Overview

* Unlocking the promise of mRNA therapeutics, E. Rohner et al, Nature Biotechnology, 2022 [[link](https://www.nature.com/articles/s41587-022-01491-z)] [review]

  ### Clincial trials

* Safety and Efficacy of the BNT162b2 mRNA Covid-19 Vaccine, F. Polack et al, The New England Journal of Medicine, 2020 [[link](https://www.nejm.org/doi/full/10.1056/nejmoa2034577)]
* Efficacy and Safety of the mRNA-1273 SARS-CoV-2 Vaccine, L. Baden et al, The New Egnland Journal of Medicine, 2020 [[link](https://www.nejm.org/doi/full/10.1056/nejmoa2035389)]

## mRNA translation and stability

* mRNAs, proteins and the emerging principles of gene expression control, C. Buccitelli and M. Selbach, Nature Reviews Genetics, 2020 [[link](https://www.nature.com/articles/s41576-020-0258-4)] [review]
* Combinatorial optimization of mRNA structure, stability, and translation for RNA-based therapeutics, K. Leppek et al, Nature Communications, 2022 [[link](https://www.nature.com/articles/s41467-022-28776-w)]

### Translation quantification

* Ribosome profiling reveals the what, when, where and how of protein synthesis, G. Brar and J. Weissman, Nature Reviews Molecular Cell Biology, 2015 [[link](https://www.nature.com/articles/nrm4069)] [review]
* A quantitative proteome map of the human body, L. Jiang et al, Cell, 2020 [[link](https://www.sciencedirect.com/science/article/pii/S0092867420310783)] [data]

### 5' untranslated region (UTR)

* Functional 5′ UTR mRNA structures in eukaryotic translation regulation and how to find them, K. Leppek et al, Nature Reviews Molecular Cell Biology, 2018 [[link](https://www.nature.com/articles/nrm.2017.103)] [review]
* Human 5′ UTR design and variant effect prediction from a massively parallel translation assay, P. Sample et al, Nature Biotechnology, 2019, [link](https://www.nature.com/articles/s41587-019-0164-5) [data, machine learning]
* High-throughput 5′ UTR engineering for enhanced protein production in non-viral gene therapies, J. Cao et al, Nature Communications, 2021 [[link](https://www.nature.com/articles/s41467-021-24436-7)] [data, machine learning]
* Predicting mean ribosome load for 5’UTR of any length using deep learning, A. Karollus et al, PLoS Computational Biology, 2021 [[link](https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1008982)] [data, machine learning]
### Codon optimization

* Codon optimality, bias and usage in translation and mRNA decay, G. Hanson and J. Coller, Nature Reviews Molecular Cell Biology, 2018 [[link](https://www.nature.com/articles/nrm.2017.91)] [review]
* Algorithm for optimized mRNA design improves stability and immunogenicity, H. Zhang, arXiv, 2020 [[link](https://arxiv.org/abs/2004.10177)] [machine learning, software]
* Codon optimization with deep learning to enhance protein expression, H. Fu et al, Scientific Reports, 2020 [[link](https://www.nature.com/articles/s41598-020-74091-z)] [machine learning]
* mRNAid, an Open-Source Platform for Therapeutic mRNA Design and Optimization Strategies, N. Vostrosablin, bioRxiv, 2022 [[link](https://www.biorxiv.org/content/10.1101/2022.04.04.486952v1)] [software]

### 3' UTR

* What are 3' UTRs doing?, C. Mayr, Cold Spring Harbor Perspectives in Biology, 2019 [[link](https://cshperspectives.cshlp.org/content/11/10/a034728.long)] [review]

## mRNA structure

* Advances and opportunities in RNA structure experimental determination and computational modeling, J. Zhang et al, Nature Methods, 2022 [[link](https://www.nature.com/articles/s41592-022-01623-y)] [review]
### secondary structure

* Secondary structure prediction for RNA sequences including N6-methyladenosine, E. Kierzek et al, Nature Communications, 2022 [[link](https://www.nature.com/articles/s41467-022-28817-4)] [machine learning, software]
* RNA secondary structure packages evaluated and improved by high-throughput experiments, H. Wayment-Steele, Nature methods, 2022 [[link](https://www.nature.com/articles/s41592-022-01605-0)] [data, machine learning]

### 3D structure

* E2Efold-3D: End-to-End Deep Learning Method for accurate de novo RNA 3D Structure Prediction, T. Shen, arXiv, 2022 [[link](https://arxiv.org/abs/2207.01586)] [machine learning]


## Delivery technology

* RNA-responsive elements for eukaryotic translational control, E. Zhao et al, Nature Biotechnology, 2021 [[link](https://www.nature.com/articles/s41587-021-01068-2)]
* Programmable RNA sensing for cell monitoring and manipulation, Y. Qian et al, Nature, 2022 [[link](https://www.nature.com/articles/s41586-022-05280-1)]
* Modular, programmable RNA sensing using ADAR editing in living cells, K. Kaseniit et al, Nature Biotechnology, 2022 [[link](https://www.nature.com/articles/s41587-022-01493-x)]
* Programmable eukaryotic protein synthesis with RNA sensors by harnessing ADAR, K. Jiang et al, Nature Biotechnology, 2022 [[link](https://www.nature.com/articles/s41587-022-01534-5)]

## mRNA production

* An engineered T7 RNA polymerase that produces mRNA free of immunostimulatory byproducts, A. Dousis, Nature Biotechnology, 2022 [[link](https://www.nature.com/articles/s41587-022-01525-6)]

## circular RNA (including cap-independent mRNA translation)

* Engineering circular RNA for enhanced protein production, R. Chen et al, Nature Biotechnology, 2022 [[link](https://www.nature.com/articles/s41587-022-01393-0)]

### Internal ribosome entry site (IRES)

* Systematic discovery of cap-independent translation sequences in human and viral genomes, S. Weingarten-Gabbay et al, Science, 2016 [[link](https://www.science.org/doi/10.1126/science.aad4939)] [data]
* Structured elements drive extensive circular RNA translation, C. Chen et al, Molecular Cell, 2021 [[link](https://www.sciencedirect.com/science/article/pii/S1097276521006262)] [data]
* Pervasive translation of circular RNAs driven by short IRES-like elements, X. Fan et al, Nature Communications, 2022 [[link](https://www.nature.com/articles/s41467-022-31327-y)] [data]

